### PR TITLE
Solution for problem statement 1 - Vignesh

### PR DESCRIPTION
**Pull request by vigneshk08**

### CODE REVIEW

File: Solution_problem1.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
+import java.io.BufferedReader;
```
Comment (maknahar) [inline Solution_problem1.java]: Hi Vignesh, Thanks for submitting the solution. Could you please add a Readme explaining the dependencies, HOWTORUN and other details?

**Comment by vigneshk08** [comment]
I have committed the readme file for your reference. Please check.

File: Solution_problem1.java [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+				}
+			}
+			FinalCalculate(Delivery,size,filtersize,writer);
+		}catch(Exception e) {
+			e.printStackTrace();
+		}
+	
+		}
+	
+	private static void FinalCalculate(String Delivery,int size,int filtersize,FileWriter writer) {
```
Comment (maknahar) [inline Solution_problem1.java]: @vigneshk08 This works. 👍

Comment (maknahar) [reply Solution_problem1.java]: Please re-request the review once you have added solution for second problem as well.

**Pull request closed without merging**